content/influxdb/cloud/migrate-regions.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Migrate to an account in a new region
+description: >
+  Learn how to migrate from one InfluxDB Cloud region to another.
+---
+
+The following guide provides instructions on migrating data and resources
+from an existing InfluxDB Cloud account to a new InfluxDB account in another
+InfluxDB Cloud region.
+
+## Create a new account in a new region
+
+InfluxDB organizations are bound to a specific cloud provider and region.
+[Create a new InfluxDB Cloud account](/influxdb/cloud/sign-up/) in the region
+you want to migrate to.
+
+## Migrate Data
+
+If you want to migrate data from your current InfluxDB Cloud account to your new,
```

Comment:
this sentence is awkward - are you sending people to migration documentation? aren't you providing instructions here in this doc?

Reply:
Yes, because the migration method depends on which storage engine they want to migrate to.